live-examples/js-examples/array/array-push.html

```diff
@@ -1,15 +1,15 @@
 <pre>
-<code id="static-js">var animals = ['pigs', 'goats', 'sheep'];
+<code id="static-js">
+const animals = ['pigs', 'goats', 'sheep'];
```

Comment:
This has the effect of adding a blank line at the start of the example. It should be:

```
<code id="static-js">const animals = ['pigs', 'goats', 'sheep'];
```

(I know, this is very non-obvious).